low-level/files/opt/seagate/sspl/conf/sspl.post-install.conf.tmpl

```diff
@@ -0,0 +1,26 @@
+cortx:
+  release:
+    product: LR_R2
+server_node:
+  30512e5ae6df9f1ea02327bab45e499d:
+    bmc:
+      ip: 198.120.10.1
+      secret: Admin123
+      user: ADMIN
+    hostname: ssc-vm-2217.colo.seagate.com
+    name: srvnode-1
+    storage:
+      enclosure_id: enclosure_1
```

Comment:
Its not called out in Prov Arch ppt as needed for post-install, do we need it here? If yes, update in ppt too

Reply:
done.